package.json

```diff
@@ -70,6 +70,7 @@
     "node-fetch": "^2.6.9",
     "platform": "^1.3.6",
     "replace-homedir": "^2.0.0",
+    "safe-json-utils": "^1.1.1",
```

Comment:
I don't see you ever using this or lokijs dependency. Are you sure they both are necessary? If all or some are not please remove and check it's still working

Reply:
I removed safe-json-utils. 

lokijs needs for WalletConnect

Reply:
https://docs.walletconnect.com/2.0/javascript/guides/nodejs